src/mito2/src/region.rs

```diff
@@ -119,6 +119,9 @@ pub(crate) struct MitoRegion {
     last_compaction_millis: AtomicI64,
     /// Provider to get current time.
     time_provider: TimeProviderRef,
+    /// The highest watermark of remote WAL since last flushing.
+    /// Sends this to meta-srv for active remote WAL pruning.
+    pub(crate) high_watermark_since_flush: AtomicU64,
```

Comment:
```suggestion
    /// The topic's latest entry id since the region's last flushing.
    /// **Only used for remote WAL pruning.**
    /// 
    /// The value will be updated to the high watermark of the topic
    /// if region receives a flush request or schedules a periodic flush task 
    /// and the region's memtable is empty.    
    ///
    /// There are no WAL entries in range [flushed_entry_id, latest_entry_id_since_flush] for current region,
    /// which means these WAL entries maybe able to be pruned up to `latest_entry_id_since_flush`.
```

Reply:
Please explain what the watermark is or what the highest watermark is.
Is it identical to the `latest_entry_id_since_flush`?

If so, can we rename this field to `topic_latest_entry_id`?